docs/HISTORY.txt

```diff
@@ -5,7 +5,8 @@ Changelog
 1.9.2 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Make ``collective.deletepermission`` dependency optional.
+  Use the extas ``ftw.lawgiver [deletepermission]`` when upgrading. [jone]
```

Comment:
maybe also rephrase a bit, i first got confused and assumed this was something that changed within this PR, but only after some seconds of staring at it i figured out that it is meant as a word of warning. maybe something like the following suggestion would clarify:

```
If you upgrade to this version of ``ftw.lawgiver`` you most likely want to enable the  ``[deletepermission]`` extra.
```